torchci/lib/bot/autoLabelBot.ts

```diff
@@ -283,13 +296,28 @@ function isNotUserFacing(filesChanged: string[]): boolean {
   );
 }
 
-function getLabelsToAddFromTitle(
+function getLabelsToAddFromIssueTitle(
   title: string,
   labelFilter: RegExp = /.*/
 ): string[] {
   const labelsToAdd: string[] = [];
 
-  for (const [regex, label] of titleRegexToLabel) {
+  for (const [regex, label] of IssueTitleRegexToLabel) {
+    if (title.match(regex) && label.match(labelFilter)) {
+      labelsToAdd.push(label);
+    }
+  }
+
+  return labelsToAdd;
+}
```

Comment:
Maybe a helper function or something that takes in a list to reduce code duplication?